src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -2015,22 +2015,25 @@ tools to limit workflow activity.
 Runahead Limiting
 ^^^^^^^^^^^^^^^^^
 
-The scheduler runahead limit determines how many consecutive cycle points can
-be active at once. The base point of the runahead calculation is the current
-lowest-valued point with :term:`active` or :term:`incomplete` tasks present.
+The runahead limit prevents a workflow from getting too far ahead of the oldest
+active cycle point.
 
-The runahead limit can be set with :cylc:conf:`[scheduling]runahead limit`,
-to an integer interval (for datetime or integer cycling workflows) or to
-a datetime interval (datetime cycling only).
+A cycle is considered to be active if it contains any :term:`active` or
+:term:`incomplete` tasks.
+
+The runahead limit can be configured for a workflow by
+:cylc:conf:`[scheduling]runahead limit`. It defines the span of cycles which
+are permitted to run as measured from the oldest active cycle point. The
+runahead limit can be defined in two ways, as an integer or datetime span.
```

Comment:
```suggestion
:cylc:conf:`[scheduling]runahead limit`. It can be defined in two ways, as an integer or datetime interval.
```
That repeats the information just above.